extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -111,17 +112,18 @@ require(['wikia.window', 'jquery', 'wikia.nirvana', 'wikia.tracker', 'JSMessages
 							$this.addError(tooLongLabelError);
 							return true;
 						}
-						// check if value already exists (in cachedVals variable)
-						if (cachedVals.indexOf(val) === -1) {
-							// not exists, add it to cachedVals and remove previous errors
-							cachedVals.push(val);
-
-							$this.removeError();
```

Comment:
Return just goes for next `each` iteration, and error is added before each `return` statement. It's intended.